src/components/shared/BannerSection.vue

```diff
@@ -1,32 +1,30 @@
 <script setup>
 import Logo from '@/assets/icons/LogoIcon.svg';
 import GenericLink from '@/components/generics/GenericLink.vue'
-import { defineProps } from '@vue/runtime-core'
 
 const props = defineProps({
     navName: String
 });
 </script>
 
 <template>
-    <div class="navCard">
-        <div class="navCard__items">
-            <generic-link href="#" container-class="navigation__logo-link" :text-first="false">
+    <div class="banner">
+        <div class="banner__items">
+            <generic-link href="/" container-class="navigation__logo-link" :text-first="false">
                 <Logo />
             </generic-link>
-            <p class="navCard__name-main">{{ props.navName }}</p>
-            <div class="navCard__path">
-                <generic-link href="#" container-class="navigation__logo-link" text-content="Home" :text-first="false"
-                    class="navCard__link" />
-                <span class="navCard__arrow">></span>
-                <span>{{ props.navName }}</span>
+            <p class="banner__name-main text-3xl">{{ props.navName }}</p>
+            <div class="banner__path">
+                <generic-link href="/" container-class="navigation__logo-link" text-content="Home" :text-first="false" class="banner__link text-md" />
```

Comment:
review the classes for generic links container class should be the only one, no additional native 'class' atribute